clippy_lints/src/significant_drop_tightening.rs

```diff
@@ -421,11 +421,10 @@ fn dummy_stmt_expr<'any>(expr: &'any hir::Expr<'any>) -> hir::Stmt<'any> {
 }
 
 fn has_drop(expr: &hir::Expr<'_>, first_bind_ident: &Ident, lcx: &LateContext<'_>) -> bool {
-    if let hir::ExprKind::Call(fun, args) = expr.kind
+    if let hir::ExprKind::Call(fun, [first_arg, ..]) = expr.kind
```

Comment:
While we're already here, can this also just be `[first_arg]` given that this is matching `mem::drop` or is there a specific reason you left this out?

Reply:
Done, just missed the mem_drop check.